File: game-server/src/main/java/org/forfun/mmorpg/game/cross/message/RpcHeartBeat.java
package org.forfun.mmorpg.game.cross.message;

import jforgame.socket.share.annotation.MessageMeta;
import jforgame.socket.share.message.Message;
import lombok.Getter;
import lombok.Setter;
import org.forfun.mmorpg.game.cross.constant.ConstantCross;

@Getter
@Setter
@MessageMeta(cmd = ConstantCross.CMD_HEART_BEAT)
public class RpcHeartBeat implements Message {
}

File: game-server/src/main/kotlin/org/forfun/mmorpg/game/cross/message/RpcHeartBeat.kt
package org.forfun.mmorpg.game.cross.message

import jforgame.socket.share.annotation.MessageMeta
import jforgame.socket.share.message.Message
import org.forfun.mmorpg.game.cross.constant.ConstantCross

@MessageMeta(cmd = ConstantCross.CMD_HEART_BEAT)
class RpcHeartBeat : Message
